fossunited/foss_hackathon/doctype/foss_hackathon_team/foss_hackathon_team.py

```
@@ -130,26 +143,58 @@ def has_permission(self, ptype="read", user=None):
+    """Helper function to delete team"""
+    try:
+        if frappe.db.exists(HACKATHON_TEAM, team_name):
             frappe.delete_doc(
                 HACKATHON_TEAM,
-                self.name,
+                team_name,
                 ignore_permissions=True,
                 force=True,
             )
+    except Exception:
+        pass
```

Ruff (0.15.5)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)